src/auth/strategies/payloads/two-factor-auth.payload.ts

```diff
@@ -0,0 +1,5 @@
+export interface TwoFactorAuthPayload {
+  id: number;
+  status: string;
+  message: string;
```

Comment:
as I said in some other comment, I would remove `status` and `message` from the token payload